src/actions/readingStats.js

```diff
@@ -0,0 +1,51 @@
+import axios from 'axios';
+import { FETCH_READING_STATS_SUCCESS, FETCH_READING_STATS_REQUEST, FETCH_READING_STATS_FAILED } from '../actionTypes/readingStats';
+
+export const readingStatsRequesting = state => ({
+  type: FETCH_READING_STATS_REQUEST,
+  payload: state,
+})
+
+export const readingStatsError = err => ({
+  type: FETCH_READING_STATS_FAILED,
+  payload: err,
+});
+
+export const showReadingStats = data => ({
+  type: FETCH_READING_STATS_SUCCESS,
+  payload: data,
+});
+
+const API = process.env.REACT_APP_API;
+
+export const getReadingStats = () => dispatch => {
+  dispatch(readingStatsRequesting(true));
+  const { token } = window.localStorage;
```

Comment:
This can also be refactored to 
```suggestion
  const { token } = localStorage;
```